tests/sesolve/closed_system.py

```diff
@@ -194,8 +197,8 @@ def grads_expect(self, t: float) -> Tensor:
         ]).detach()
 
 
-cavity = Cavity(n=8, delta=2 * pi, alpha0=1.0, t_end=1.0)
-gcavity = Cavity(n=8, delta=2 * pi, alpha0=1.0, t_end=1.0, requires_grad=True)
+cavity = Cavity(n=8, delta=2.0, alpha0=0.5, t_end=1.0)
+gcavity = Cavity(n=8, delta=2.0, alpha0=0.5, t_end=1.0, requires_grad=True)
```

Comment:
Nit: delta is in radians so conventionally we express it in fractions of `2π`

Reply:
Right, I changed it to avoid having `t_end` coincinding with a full rotation, which makes a lot of the gradient we check cancel, and was hiding sign errors. This commit fixes this more clearly and restablishes the $2\pi$ unit.